src/systems/audio-system.js

```diff
@@ -84,4 +97,25 @@ export class AudioSystem {
     document.body.addEventListener("touchend", resume, false);
     document.body.addEventListener("mouseup", resume, false);
   }
+
+  addStreamToOutboundAudio(id, mediaStream) {
+    if (this.audioNodes.has(id)) {
+      this.removeStreamFromOutboundAudio(id);
+    }
+
+    const sourceNode = this.outboundContext.createMediaStreamSource(mediaStream);
+    const gainNode = this.outboundContext.createGain();
+    sourceNode.connect(gainNode);
+    gainNode.connect(this.outboundGainNode);
+    this.audioNodes.set(id, { sourceNode, gainNode });
+  }
+
+  removeStreamFromOutboundAudio(id) {
+    if (this.audioNodes.has(id)) {
+      const nodes = this.audioNodes.get(id);
+      nodes.sourceNode.disconnect();
+      nodes.gainNode.disconnect();
+      this.audioNodes.delete(id);
+    }
+  }
 }
```

Comment:
Sorry, another comment unrelated to the lines changed in this PR... 

Here we call `setLocalDescription` and `setRemoteDescription`, which both return promises. Should we be `await`ing on them (or using `.then`)?

https://github.com/mozilla/hubs/blob/1477448a4576eb693b31c9004fbdd4eb39a30c1d/src/systems/audio-system.js#L46